docs/intro.md

```diff
@@ -0,0 +1,127 @@
+# A brieft introduction to HADDOCK3
+
+HADDOCK3 is the next generation of integrative modelling software in the
+long-lasting HADDOCK project. It represents a complete rethinking and rewriting
+of the HADDOCK2.X series, implementing a new way to interact with HADDOCK and
+offering new features to users who can now define custom workflows. 
+
+In the previous HADDOCK2.x versions, users had access to a highly
+parameterisable yet rigid simulation pipeline composed of three mandatory steps:
```

Comment:
was "water" always mandatory in 2.x?

Reply:
Not really - may-be instead of water we can rather call it: final refinement